src/types/ApiResponse.ts

```diff
@@ -0,0 +1,9 @@
+export interface ApiResponse<T> {
```

Comment:
I'm a ~~TS hater~~ TS newbie, but if this interface it's only for rick api, it shouldn't be with the api? Or at least with most specific name?

Reply:
I will rename it

Reply:
👍🏼  But next catch-up we'll discuss about where should be the types 👅